### PR TITLE
Trigger qor5-provisioning github actions

### PR DESCRIPTION
**Pull request by bohrasd**
https://theplanttokyo.atlassian.net/browse/SRE-2908

### CODE REVIEW

File: hack/build-deploy-app.sh [hunk 1]
```diff
@@ -10,5 +10,8 @@ source hack/get-tools.sh
 plantbuild push ./plantbuild/build.jsonnet
 plantbuild k8s_set_images ./plantbuild/images.jsonnet
```
Comment (idealhack) [inline hack/build-deploy-app.sh:11]: This is the actual deploy step for K8s and should be removed

File: hack/build-deploy-app.sh [hunk 2]
```diff
@@ -10,5 +10,8 @@ source hack/get-tools.sh
...
 
-NAMESPACE="qor5-test"
-echo "kubectl -n $NAMESPACE get deploy -o name | xargs -n1 kubectl -n $NAMESPACE rollout status --timeout 150s" | $KUBECTL_BASH
+curl -XPOST -H "Content-Type: application/json" \
+    -H "Accept:  application/vnd.github.everest-preview+json" \
+    -H "Authorization: Bearer $GITHUB_TOKEN" \
+    https://api.github.com/repos/theplant/qor5-provisioning/dispatches \
+    ∙ -d '{"event_type":"deploy-test","client_payload":{"github":{"example":{"image":"public.ecr.aws/qor5/example:latest"}}}}'
```
Comment (idealhack) [inline hack/build-deploy-app.sh:17]: Why is there a `∙` here? Can you add the GitHub API doc reference as a comment in code?

Comment (bohrasd) [reply hack/build-deploy-app.sh:17]: I copied from my console, it was a mistake, sorry

File: hack/build-deploy-app.sh [hunk 3]
```diff
@@ -8,7 +8,10 @@ set -o pipefail
...
-plantbuild k8s_set_images ./plantbuild/images.jsonnet
 
-NAMESPACE="qor5-test"
-echo "kubectl -n $NAMESPACE get deploy -o name | xargs -n1 kubectl -n $NAMESPACE rollout status --timeout 150s" | $KUBECTL_BASH
+# https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event
+curl -XPOST -H "Content-Type: application/json" \
+    -H "Accept:  application/vnd.github.everest-preview+json" \
+    -H "Authorization: Bearer $GITHUB_TOKEN" \
+    https://api.github.com/repos/theplant/qor5-provisioning/dispatches \
+    -d '{"event_type":"deploy-test","client_payload":{"github":{"example":{"image":"public.ecr.aws/qor5/example:latest"}}}}'
```
Comment (idealhack) [inline hack/build-deploy-app.sh:17]: I'm unsure if using an (unchanged) `latest` tag can deploy the new version, but I'll defer to you.

**Review by idealhack — APPROVED** [review]

**Pull request merged**